### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by Camilovelag**
# Good day! 🌤️

This pull request is intended to create and query multiple tables.

## 🛠️ Changes made

✔️ Create `owners` and `species` tables.
✔️ Update `animals` table.
✔️ Add queries to answer assigned questions (screenshots below).

I'm expecting feedback.

Happy coding!  👏👏👏

query1
query2

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -13,3 +13,24 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Boarmon', '2000-06-07', 20.4, true, 7);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Blossom', '1998-10-13', 17, true, 3);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Ditto', '2022-05-14', 22, true, 4);
+
+/* Insert data into owners tables */
```
Comment (HAMMAS-SALEEM) [inline data.sql:17]: Capture

- [ ] You are not following the latest version of readme file. Kindly use the latest version of readme file using these [guidlines](https://github.com/microverseinc/readme-template/blob/master/README.md) and fill the missing things in your readme file 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Camilovelag :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Camilovelag) [reply data.sql:17]: Hi @HAMMAS-SALEEM 👋 
I'm using the README template specified for this [project](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md). It is needed to change de README file after all?
Thanks!

P.S.: I face the same situation in the last [approved PR](https://github.com/Camilovelag/Vet-clinic-database/pull/2) 😵‍💫

Comment (sja-thedude) [reply data.sql:17]: Hey @Camilovelag,
You have to basically use the new readme template given [here](https://github.com/microverseinc/readme-template#readme). If you could update in the next 5-10 mins then I will approve else again ask for changes.

Comment (HAMMAS-SALEEM) [reply data.sql:17]: Hey @Camilovelag i hope you are doing well.

I don't know why they specified the old readme template but it had been objectively said to us that the latest version of readme will be used in the upcoming projects 😊 

For further you can ask for a second opinion where microverse team will see this matter and they would do fair or if this is Microverse fault then they would update their materials 👍 

You can create an issue in that activity about this thing 😊

Comment (HAMMAS-SALEEM) [reply data.sql:17]: Hey @Camilovelag I hope you are doing well. I can see that they have updated the readme file and there is a link to your new readme file in this [project](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md) document 👍

## Click on the link ⬇️
Capture1
## Click on the link ⬇️
Capture2

Comment (Camilovelag) [reply data.sql:17]: Done! 😄

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Camilovelag, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Camilovelag** [comment]
Thanks @sja-thedude @HAMMAS-SALEEM  👍

**Pull request merged**